index.js

```diff
@@ -0,0 +1,77 @@
+import { displayBooksControl, removeBook, addBook } from './modules/display.js';
+import {
+  displayBooks, displayForm, contact, displayContact,
+  showBooksOnly, showOnlyContact, showOnlyForm,
+} from './modules/view.js';
+
+import { DateTime } from './modules/luxon.js';
+
+const bookItems = document.querySelector('#book-list');
+const awesomeHeading = document.querySelector('#awesome');
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// display  all the books
+window.onload = displayBooksControl.displayBooks();
+
+// add books by inputting into form
+
+const form = document.querySelector('form');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  // validating form fields
+  if (title === '' || author === '') {
+    // eslint-disable-next-line no-alert
+    alert('You must fill in all fields');
```

Comment:
- [ ] Kindly don't use alert because it is against the JavaScript best practices :thinking: You can make custome modals to show the validations :blush: